docs/src/main/asciidoc/init-tasks.adoc

```diff
@@ -0,0 +1,83 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Initialization tasks
+:categories: initialization
+:summary: This guide explains how to configure initialization tasks
+
+There are often initialization tasks performed by Quarkus extensions that are meant to be run once.
+For example Flyway or Liquibase initialization fall into that category. But what happens when the scaling
+needs of an application requires more instances of the application to run? Or what happens when the application
+restarts ?
+
+A common environment where both of these cases are pretty common is Kubernetes. To address these challenges,
+Quarkus allows externalization of such tasks as Kubernetes Jobs and uses init containers to ensure that an 
+application instance only starts once the initialization jobs have finished. With this approach even if an 
+application has multiple replicas, the initialization logic will only run once.
+
+This approach is reflected in the manifests generated by xref:kubernetes.adoc[Kubernetes extension].
+
+== Disabling the feature
+
+The feature can be explictily disabled per task (enabled by default).
+The default behavior can change setting the following property to `false`:
+
+[source,properties]
+----
+quarkus.openshift.init-task-defaults.enabled=false
+----
+
+In the case where we need to disable a particular task, we can use the following property:
+
+[source,properties]
+----
+quarkus.openshift.init-tasks."<task name>".enabled=false
+----
+
+The task name is the name of the Kubernetes Job resource that follows the convention `${quarkus.appliction.name}-${extension}-init`.
+If the convention is hard to remember you can always peek at the generated manifests.
```

Comment:
This uses the generated job name for correlation. The job names needs to be unique and thus needs both the app name and the extension. 

I agree with your that it's too complicated and we need to change that. I'll change the implementation so that it just uses the extension name and not the full job name.